servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -80,8 +74,10 @@ def get_job(id):
     Returns:
         JobMetadataResponse: Response containing relevant metadata
     """
-    proj_id, job_id, task_id = job_ids.api_to_dsub(id, _provider_type())
-    provider = providers.get_provider(_provider_type(), proj_id, _auth_token())
+    proj_id, job_id, task_id = job_ids.api_to_dsub(id,
+                                                   generator.provider_type())
+    provider = providers.get_provider(generator.provider_type(), proj_id,
```

Comment:
Is this due to changes in yapf? Or your IDE? If it's the latter, please turn off the autoformatting or else make it match our yapf formatter

Reply:
It is due to yapf.